fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -160,8 +162,78 @@ def test_delete_an_object_from_a_container(self):
+
+        ## Creating a new big object
+        import urllib2
+        f = urllib2.urlopen(BIG_FILE_URL_1)
+        data = f.read()
+        with open(SWIFT_RESOURCES_PATH + TEST_BIG_OBJECT_PREFIX + TEST_BIG_FILE_EXTENSION, "wb") as code:
+            code.write(data)
+
+        ## Uploading the object
+        try:
+            response = self.swift_operations.create_object(containerName, SWIFT_RESOURCES_PATH
+                                                                + TEST_BIG_OBJECT_PREFIX + TEST_BIG_FILE_EXTENSION,
+                                                                big_object_name)
+            origin = hashlib.md5(open(SWIFT_RESOURCES_PATH + TEST_BIG_OBJECT_PREFIX + TEST_BIG_FILE_EXTENSION, 'rb')
+                              .read()).hexdigest()
+
+            self.assertIsNotNone(response, "Object could not be created")
+            self.test_world['swift_objects'].append(containerName + "/" + big_object_name)
+            self.logger.debug("Created %s object was created", big_object_name)
+        except Exception as ex:
+            self.logger.error("Object %s has not been uploaded: ", big_object_name)
+            self.fail(ex)
+
+        ## Downloading the object
+        try:
+            response = self.swift_operations.get_object(containerName, big_object_name,
+                                                             SWIFT_RESOURCES_PATH)
+            self.assertTrue(response, "Object could not be downloaded")
+
+            remote = hashlib.md5(open(SWIFT_RESOURCES_PATH + big_object_name, 'rb')
+                              .read()).hexdigest()
+        except Exception as ex:
+            self.logger.error("Object %s has not been downloaded: ", big_object_name)
+            self.fail(ex)
+
+        ## Checking original file with remote file
+        self.assertEqual(origin, remote, "The original file and the downloaded file are different")
+
+        ##
+        suffix2 = datetime.utcnow().strftime('%Y%m%d%H%M%S%m')
+        f2 = urllib2.urlopen(BIG_FILE_URL_2)
+        data = f2.read()
+        with open(SWIFT_RESOURCES_PATH + TEST_BIG_OBJECT_PREFIX + suffix2 + TEST_BIG_FILE_EXTENSION, "wb") as code2:
+            code2.write(data)
+
+        ## Checking that two different files generates are different
+        origin2 = hashlib.md5(open(SWIFT_RESOURCES_PATH + TEST_BIG_OBJECT_PREFIX + suffix2 + TEST_BIG_FILE_EXTENSION,
+                                   'rb')
+                              .read()).hexdigest()
+
+        self.assertNotEqual(origin2, remote, "The second file and the downloaded file are the same, "
+                                             "it cannot be possible")
+
+        os.remove(SWIFT_RESOURCES_PATH + big_object_name)
+        os.remove(SWIFT_RESOURCES_PATH + TEST_BIG_OBJECT_PREFIX + TEST_BIG_FILE_EXTENSION)
+        os.remove(SWIFT_RESOURCES_PATH + TEST_BIG_OBJECT_PREFIX + suffix2 + TEST_BIG_FILE_EXTENSION)
```

Comment:
Should we put this code in the "cleanUp" procedure? If not, what will happen if some exception is raised between the file creation and this lines?